bukkit/src/main/java/net/william278/huskhomes/user/BukkitUser.java

```diff
@@ -25,10 +25,14 @@
 import net.william278.huskhomes.position.Position;
 import net.william278.huskhomes.teleport.TeleportationException;
 import net.william278.huskhomes.util.BukkitAdapter;
+import net.william278.huskhomes.util.BukkitTask;
+import net.william278.huskhomes.util.Task;
+import org.bukkit.Bukkit;
 import org.bukkit.entity.Player;
 import org.bukkit.event.player.PlayerTeleportEvent;
 import org.bukkit.metadata.MetadataValue;
 import org.bukkit.permissions.PermissionAttachmentInfo;
+import org.bukkit.plugin.Plugin;
```

Comment:
Could you auto-cleanup your imports? (If you're using intellij, select the root project and use CTRL+ALT+O)

Reply:
Yeah i thought i had auto clean up enabled appears not